tmk_core/common/eeconfig.h

```diff
@@ -37,12 +37,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define EECONFIG_UNICODEMODE                        (uint8_t *)12
 #define EECONFIG_STENOMODE                          (uint8_t *)13
 // EEHANDS for two handed boards
-#define EECONFIG_HANDEDNESS         			        	(uint8_t *)14
+#define EECONFIG_HANDEDNESS                         (uint8_t *)14
 #define EECONFIG_KEYBOARD                          (uint32_t *)15
 #define EECONFIG_USER                              (uint32_t *)19
 #define EECONFIG_VELOCIKEY                          (uint8_t *)23
 
-#define EECONFIG_HAPTIC                            (uint32_t*)24
+#define EECONFIG_HAPTIC                             (uint32_t*)24
+#define EECONFIG_RGB_MATRIX                         (uint8_t *)28 // 5 bytes, next address is 33
```

Comment:
Honestly, I would prefer that this accurately represents the range that we're using. 

```suggestion
#define EECONFIG_RGB_MATRIX                         (uint32_t *)28 
#define EECONFIG_RGB_MATRIX_SPEED                   (uint8_t *)32   
```

Reply:
approved